src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -453,4 +461,174 @@ void GIVEN_system_config_with_small_periodic_interval_WHEN_active_logs_included_
         assertEquals(1, allFiles.size());
         logManagerService.ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG.set(false);
     }
+
+    @Test
+    void GIVEN_files_randomly_named_WHEN_all_files_renamed_randomly_THEN_all_files_only_send_once()
+            throws Exception {
+        //TODO: Intentionally pass the test now, since it will fail for current code.
+        if (!ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG.get()) {
```

Comment:
This is weird. Now you are letting this test be controlled from the outside. You should keep doing what you are doing with the other ones where the test sets the flag to a value

Reply:
This is for skipping this tests, because the PR just try to show the newly tests, but this test is not ran until next PR.

Reply:
Still a bad idea. you are basically having something external that is not being set by this test execute it. If this is related to the active log file why cant you just set the `logManagerService.ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG.set(true);` and then back to false

Reply:
This is not just related to active log file, it is also related to the log rotation, which the current logic will not handle. So main idea is that skip this test.

Reply:
I you set the active log file flag to true on another test this test would run. I don't see how that is skipping this test. You are operating under the assumption that this is false. Additionally what is the value of a test if you don't want to run it? - Regardless if you really want to skip the test for now there is an annotation called @Disabled that you can use